proxy/src/bin/proxy.rs

```diff
@@ -697,22 +730,45 @@ fn build_auth_backend(
             )?));
             tokio::spawn(locks.garbage_collect_worker());
 
-            let url = args.auth_endpoint.parse()?;
+            let url: proxy::url::ApiUrl = args.auth_endpoint.parse()?;
```

Comment:
I was thinking we add a new auth_backend variant (so we can replace "console", but also to make the update more explicit). wdyt?

Reply:
"cplane-v1" (since "cplane" is an alias for console) with "control-plane" as an alias is fine by you?